api/v1beta1/clusterclass_types.go

```diff
@@ -809,12 +824,14 @@ type ValidationRule struct {
 	// skipped if an old value could not be found.
 	//
 	// +required
+	// +kubebuilder:validation:MaxLength=512
```

Comment:
CEL rules can be very long, I'd probably make this longer, 4096?

Reply:
Will do